--- pyproject.toml.orig	2026-01-19 12:56:26.794995000 -0400
+++ pyproject.toml	2026-01-19 12:56:59.133233000 -0400
@@ -118,13 +118,10 @@
     "jsonschema>=3.2.0,<4.30.0",
 ]
 
-[project.scripts]
-cps = "calibreweb:main"
-
 [tool.setuptools]
 include-package-data = true
 license-files = ["LICENSE"]
+py-modules = []
 
 [tool.setuptools.dynamic]
-version = {attr = "calibreweb.cps.constants.STABLE_VERSION"}
-
+version = {attr = "cps.constants.STABLE_VERSION"}
